music_album.rb

```diff
@@ -1,16 +1,26 @@
 require_relative 'item'
 
 class MusicAlbum < Item
-  attr_accessor :on_spotify
+  attr_accessor :name, :on_spotify
 
-  def initialize(on_spotify)
+  def initialize(name, on_spotify)
     super(genre, author, label, publish_date)
+    @name = name
     @on_spotify = on_spotify
     @archived = archived
   end
 
   def can_be_archived?
-    return true if @archived || @on_spotify == true
+    return true if @archived && @on_spotify == true
```

Comment:
The operator was changed to || because I have rechecked the requirements I found this `should return true if parent's method returns true AND if on_spotify equals true` I suggest that we could use &&

Reply:
Sorry, my mistake.